Parking/OnStreetParking/doc/spec.md

```diff
@@ -152,6 +165,18 @@ This value must aggregate free spots from all groups devoted to special parking
     + Default unit: Meters
     + Optional
 
++ `maximumAllowedHeight` : Maximum allowed height for vehicles. If there are multiple zones, it will be the minimum height of
+all the zones.
+    + Attribute type: [Number](http://schema.org/Number)
+    + Default unit: Meters
+    + Optional
+
```

Comment:
[This](https://www.google.es/maps/@37.4821373,-5.9477948,3a,68.8y,28.79h,73.78t/data=!3m6!1e1!3m4!1sUntOnrc3lILkhPmlfQn6-g!2e0!7i13312!8i6656?hl=en) is my town, we can park there on weekday mornings.